Example/Tests/NSAttributeStringAddsTests.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  NSAttributeStringAddsTests.swift
+//  LithoStringsTests
+//
+//  Created by Remmington Damper on 1/31/22.
+//
+
+import Foundation
+import UIKit
+@testable import LithoStrings
+import XCTest
+import SwiftUI
+
+class NSAttributeStringAddsTests: XCTestCase {
+    
+func testSetForegroundColorAttributes() {
```

Comment:
This need to be indented. Anything that's enclosed with brackets should be indented one tab from the start of the line of its 'parent bracket'. Since these functions are within a class, they need to be indented.